hunting/windows/docs/detect_masquerading_attempts_as_native_windows_binaries.md

```diff
@@ -0,0 +1,39 @@
+# Detect masquerading attempts as native Windows binaries
+
+---
+
+## Metadata
+
+- **Author:** Elastic
+- **UUID:** `93a72542-a1f7-4407-9175-8f066343db60`
+- **Integration:** [endpoint](https://docs.elastic.co/integrations/endpoint)
+- **Language:** `ES|QL`
+
```

Comment:
We are missing descriptions for these.

Example -> https://github.com/elastic/ia-trade-team/issues/302#issuecomment-1952390789

```
Adversaries may attempt to manipulate features of their artifacts to make them appear legitimate or benign to users and/or security tools. Masquerading occurs when the name or location of an object, legitimate or malicious, is manipulated or abused for the sake of evading defenses and observation. This hunt detects processes named as legit Microsoft native binaries located in the system32 folder.
```

Reply:
I added support for this in https://github.com/elastic/detection-rules/pull/3642/commits/4b4a8f384d7df730b482c5a440761b0764b5bec6

The hunts will need to be updated to split out the description - once done, I can regenerate the docs. It will be failing until the descriptions are added

Reply:
upon further discussion, the description decoupling will happen in a follow on PR - rolled back